docker-compose-intersection.yml

```diff
@@ -63,89 +64,9 @@ services:
       cvmanager_keycloak:
         condition: service_healthy
         required: false
-      mongodb_container:
+      mongo:
         condition: service_healthy
         required: false
       kafka-setup:
         condition: service_started
         required: false
-
-  mongodb_container:
-    profiles:
-      - intersection
-      - intersection_no_api
-      - conflictmonitor
-      - mongodb_container
-    image: mongo:8
-    container_name: jpo-conflictmonitor-mongodb-container
-    restart: always
-    environment:
-      - MONGO_REPLICA_SET_NAME=rs0
-      - DB_HOST_IP=${DB_HOST_IP}
-      - MONGO_INITDB_ROOT_USERNAME=${MONGO_INITDB_ROOT_USERNAME}
-      - MONGO_INITDB_ROOT_PASSWORD=${MONGO_INITDB_ROOT_PASSWORD}
-      - CM_MONGO_CONNECTOR_USERNAME=${CM_MONGO_CONNECTOR_USERNAME}
-      - CM_MONGO_CONNECTOR_PASSWORD=${CM_MONGO_CONNECTOR_PASSWORD}
-      - CM_MONGO_API_USERNAME=${CM_MONGO_API_USERNAME}
-      - CM_MONGO_API_PASSWORD=${CM_MONGO_API_PASSWORD}
-      - CM_MONGO_USER_USERNAME=${CM_MONGO_USER_USERNAME}
-      - CM_MONGO_USER_PASSWORD=${CM_MONGO_USER_PASSWORD}
-      - CM_DATABASE_NAME=${CM_DATABASE_NAME}
-      - CM_DATABASE_STORAGE_COLLECTION_NAME=${CM_DATABASE_STORAGE_COLLECTION_NAME}
-      - CM_DATABASE_SIZE_GB=${CM_DATABASE_SIZE_GB}
-      - CM_DATABASE_SIZE_TARGET_PERCENT=${CM_DATABASE_SIZE_TARGET_PERCENT}
-      - CM_DATABASE_DELETE_THRESHOLD_PERCENT=${CM_DATABASE_DELETE_THRESHOLD_PERCENT}
-      - CM_DATABASE_MAX_TTL_RETENTION_SECONDS=${CM_DATABASE_MAX_TTL_RETENTION_SECONDS}
-      - CM_DATABASE_MIN_TTL_RETENTION_SECONDS=${CM_DATABASE_MIN_TTL_RETENTION_SECONDS}
-      - CM_DATABASE_COMPACTION_TRIGGER_PERCENT=${CM_DATABASE_COMPACTION_TRIGGER_PERCENT}
-      - INTERSECTION_API_ENABLE_API=${INTERSECTION_API_ENABLE_API}
-      - INTERSECTION_API_ENABLE_EMAILER=${INTERSECTION_API_ENABLE_EMAILER}
-      - INTERSECTION_API_ENABLE_REPORTS=${INTERSECTION_API_ENABLE_REPORTS}
-      - INSERT_SAMPLE_DATA=${INSERT_SAMPLE_DATA}
```

Comment:
We should remove this INSERT_SAMPLE_DATA variable from the sample.env file if it is no longer used.

Reply:
Removed!

Reply:
@dmccoystephenson can we add documentation on the `MONGO_SAMPLE_DATA_RELATIVE_PATH` variable in jpo-utils to instruct the user how to not startup with sample data? It seems like if that variable is set to nothing, then it will skip the import.

Reply:
That looks great!